### DISCUSSION TITLE
database notification doesn't work (localhost)

### DISCUSSION THREAD

**Discussion by sh4msi** [filamentphp/filament#4957] [Help]
hi,
I did all the initial configuration and the notification icon is displayed next to the top menu
But when I send the notification, it is not saved in the database (notification table).
And database notification actually doesn't work for me

(**This mode only occurs on localhost** and does not give any error and works well on the server)

```
 Notification::make()
    ->title('Saved successfully')
    ->sendToDatabase(auth()->user());
```

**Comment by zepfietje** [comment]
Hard to help without any logs or reproduction repository.

**Reply by sh4msi** [reply]
Unfortunately, I cannot put the project codes here
Of course, setting up notifications in the filamentphp panel is not a complicated task, and that's why I don't know what to check.

If you have an idea to check a certain part, thank you so that I can check it

**Reply by zepfietje** [reply]
You could set up a minimal reproduction repository which shouldn't contain the entire project.

**Reply by danharrin** [reply]
<img width="1082" alt="Screenshot 2022-11-17 at 09 12 49" src="https://user-images.githubusercontent.com/41773797/202404701-99fa131b-b684-40c7-85e2-177d4963e0a0.png">

**Comment by LittleHans8** [comment]
I had the same problem today and I found out that the problem was that I had Horizon installed and after I ran horizon it worked fine.
